README.md

```diff
@@ -1,92 +1,80 @@
+    By default, Chore will run over all queues it detects among the required files. If you wish to change this behavior, you can use:
 
+    ```
+    # Note that you can use one of these options, not both. Chore will quit and
+    # make fun of you if both options are specified.
     --queues QUEUE1,QUEUE2... # a list of queues to process
     --except-queues QUEUE1,QUEUE2... # a list of queues _not_ to process
+    ```
 
-Note that you can use one or the other but not both. Chore will quit and make fun of you if both options are specified.
+1. Chore has many more options, which you can view by executing `bundle exec chore --help`
 
-### Tips for configuring Chore
+### Tips For Configuring Chore
 
-For Rails, it can be necessary to add the directory you store your jobs in to the eager loading path,
-found in application.rb. You likely need a similar approach for most apps using jobs, unless you places
-them into a directory that is already eager loaded by default. One example of this might be:
+For Rails, it can be necessary to add the directory you store your jobs in to the eager loading path, found in `application.rb`. You likely need a similar approach for most apps using jobs, unless you place them into a directory that is already eager loaded by your application. One example of this might be:
 
 ```ruby
 config.eager_load_paths += File.join(config.root, "app", "jobs")
 ```
 
-However, due to the way eager_load_paths works in Rails, this may only solve the issue
-in your production environment. You might also find it useful for other environments
-to throw soemthing like this in an config/initializers/chore.rb file, although you
-can choose to load the job files in any way you like:
+However, due to the way `eager_load_paths` works in Rails, this may only solve the issue in your production environment. You might also find it useful for other environments to throw something like this in an `config/initializers/chore.rb` file, although you can choose to load the job files in any way you like:
 
 ```ruby
-Dir["#{Rails.root}/app/jobs/**/*"].each do |file|
-  require file unless File.directory?(file)
-end unless Rails.env.production?
+if !Rails.env.production?
+  Dir["#{Rails.root}/app/jobs/**/*"].each do |file|
+    require file unless File.directory?(file)
+  end
+end
 ```
 
-### Producing and Consuming Jobs
+### Producing & Consuming Jobs
 
-When it comes to configuring Chore, you have 2 main use cases - as a producer of messages, or as a consumer of messages (the consumer is also able to produce messages if need be, but is running as it's own isolated instance of your application).
+When it comes to configuring Chore, you have 2 main use cases - as a producer of messages, or as a consumer of messages.
 
-For producers, you must do all of your Chore configuration in an intializer.
+**IS THIS TRUE?** => (the consumer is also able to produce messages if need be, but is running as its own isolated instance of your application).
```

Comment:
This doesn't seem correct to me, should I remove it?

Reply:
I assume this is just saying one _could_ call the `perform` of another job from within the consumer which seems plausible?

Reply:
A _worker_ can do that, but I don't see any logic in the consumer that would actually publish a message. @obrie do you happen to be able to figure out what was meant by this language?

Reply:
I think it's just talking about the consuming application, not the consumer queue implementation.

Reply:
That makes _a lot_ more sense, thank you. I will update this language.